packages/client/manifest.json

```diff
@@ -5186,5 +5186,88 @@
         "suffix": "repeater"
       }
     ]
+  },
+  "rowexplorer": {
+    "block": true,
+    "name": "Row explorer block",
+    "icon": "PersonalizationField",
+    "size": {
+      "width": 100,
+      "height": 100
+    },
+    "settings": [
+      {
+        "type": "dataSource",
+        "label": "Data",
+        "key": "dataSource",
+        "required": true
+      },
+      {
+        "type": "text",
+        "label": "Height",
+        "key": "height"
+      },
+      {
+        "section": true,
+        "name": "Cards",
+        "settings": [
+          {
+            "type": "field",
+            "label": "Search Field",
+            "key": "cardSearchField",
+            "nested": true
+          },
+          {
+            "type": "text",
+            "key": "cardTitle",
+            "label": "Title",
+            "nested": true,
+            "defaultValue": "Title"
+          },
+          {
+            "type": "text",
+            "key": "cardSubtitle",
+            "label": "Subtitle",
+            "nested": true,
+            "defaultValue": "Subtitle"
+          },
+          {
+            "type": "text",
+            "key": "cardDescription",
+            "label": "Description",
+            "nested": true,
+            "defaultValue": "Description"
+          },
+          {
+            "type": "text",
+            "key": "cardImageURL",
+            "label": "Image URL",
+            "nested": true
+          }
+        ]
+      },
+      {
+        "section": true,
+        "name": "Details",
+        "settings": [
+          {
+            "type": "text",
+            "key": "detailTitle",
+            "label": "Title",
+            "nested": true
```

Comment:
We need to remove the `nested` property here - the `schema` context is provided by the repeater, which is on the left side and is not a parent of the title, so attempting to use any nested bindings here will fail and evaluate to empty strings.

Reply:
Ah good catch, the title now has no access to bindings unfortunately, I don't know if there's a way to give it access to the context within the form block.

Reply:
We can technically do this by adding another context of type schema, but we'll need to create a weird double suffix because the form block itself needs a specific context key, and then the form block also already adds another layer of suffix to the repeater inside it... And the bigger issue here anyway is that if we add another context entry in the manifest that is also of type `schema` then we will end up generating duplicate bindings that look identical for all those schema fields, and it will be impossible to tell in the frontend which pertains to the list on the left, and which pertains to the form block on the right.

TLDR is I think we just have to accept that we can't use nested here and we can't allow this field to access the schema bindings.